Uni_life_tracker/app/src/main/java/com/generic/plannr/Controllers/PasswordValidator.java

```diff
@@ -1,26 +1,23 @@
-package com.generic.plannr;
+package com.generic.plannr.Controllers;
 
-import android.graphics.Color;
-import android.util.Patterns;
-import android.widget.TextView;
 import com.generic.plannr.Gateways.UserGateway;
 import com.google.android.material.textfield.TextInputLayout;
 
 import java.util.Objects;
 import java.util.regex.Pattern;
 
-public class Validator {
+public class PasswordValidator implements Validator {
+
     // At least one number, uppercase letter, lowercase letter and special char. Min of 6 characters
     private static final Pattern PASSWORD_REQ =
-            Pattern.compile("^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[!@#&()–\\[\\]?/\\\\*_$^+=]).{6,}$");
+            Pattern.compile("[\\w\\d!@#$%^&*()-_\\\\/\\[\\]=+]{6,}");
```

Comment:
Im not sure that regex gets all the cases because for example I can sign up with the password "dana11" and it doesn't show any errors

Reply:
should be all good